### PR TITLE
Gherkin scenarios updates for 0.12.6 release

### PR DESCRIPTION
**Pull request by radinamatic**
### Summary
WIP!

### Reviewer guidance

…

### References

----

### Contributor Checklist

PR process:

- [X] PR has the correct target branch and milestone
- [X] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by cpauya** [comment]
Hi @radinamatic - been following this [guide in the Kolibri 0.12.5 docs re import of users](https://kolibri.readthedocs.io/en/latest/manage.html?highlight=import%20csv#csv-file-structure)

Import of users worked with this command `kolibri manage importusers ./many-users.csv`.

```
C:\Python34\Scripts>kolibri manage importusers ./many-users.csv
[37mINFO     Running Kolibri with the following settings: kolibri.deployment.default.settings.base[0m
[37mINFO     Invoking command 'kolibri manage importusers ./many-users.csv'.[0m
[37mINFO     User created with username marcg in facility home with password marcg[0m
...
[37mINFO     185 users created[0m

C:\Python34\Scripts>
```

I'm now resuming to test your PR. 😸

File: integration_testing/features/admin/admin-find-user.feature [hunk 1]
```diff
@@ -1,33 +1,52 @@
...
       And I am on *Facility > Users* page
+      And there are more than 150 users in the facility
+
+  Scenario: Search by navigating pages
+    When I click the right arrow (next page) button
+    Then I am on the second page of the list of facility users
+    When I click the left arrow (previous page) button
+    Then I am back on the first page of the list of facility users
+    When I keep clicking the right arrow to find a user
+    Then see the numbers of pages correctly increasing in increments of 10
```
Comment (cpauya) [inline integration_testing/features/admin/admin-find-user.feature]: Then **I** see the numbers of pages correctly increasing in increments of 10

File: integration_testing/features/super-admin/super-admin-export-content-to-local-drive.feature [hunk 2]
```diff
@@ -3,61 +3,68 @@ Feature: Super admin exports content to local drive
...
+        And I see the total number and size of <channel> channel resources
+        And I see the value for *Drive space available*      
+        And I see the value for *Content selected* is 0
+      # Select/deselect all the topics
+      When I check the *Select all* checkbox
+      Then I see the checkboxes for all the topics are checked
+        And I see the *Export* button is active
+        And I see the values for *Content selected* increase
+      When I uncheck the *Select all* checkbox
+      Then I see the *Export* button is inactive
```
Comment (cpauya) [inline integration_testing/features/super-admin/super-admin-export-content-to-local-drive.feature]: Then I see the checkboxes for all the topics are unchecked
        And I see the *Export* button is inactive

File: integration_testing/features/super-admin/super-admin-export-content-to-local-drive.feature [hunk 3]
```diff
@@ -3,61 +3,68 @@ Feature: Super admin exports content to local drive
...
+      Then I see the *Export* button is active
+        And I see the values for *Content selected* increase
+      When I uncheck the <topic> topic checkbox
+      Then I see the *Export* button is inactive
+        And I see the values for *Content selected* is 0
+      # Select and import just one resource from a subtopic of a topic
+      When I click the <topic> topic
+      Then see the list of subtopics for the <topic> topic
+      When I click the <subtopic> subtopic
+      Then see the list of resources for the <subtopic> subtopic
```
Comment (cpauya) [inline integration_testing/features/super-admin/super-admin-export-content-to-local-drive.feature]: Then I see the list of resources for the <subtopic> subtopic

File: integration_testing/features/super-admin/super-admin-find-user.feature [hunk 4]
```diff
@@ -1,33 +1,52 @@
...
       And I am on *Facility > Users* page
+      And there are more than 150 users in the facility
+
+  Scenario: Search by navigating pages
+    When I click the right arrow (next page) button
+    Then I am on the second page of the list of facility users
+    When I click the left arrow (previous page) button
+    Then I am back on the first page of the list of facility users
+    When I keep clicking the right arrow to find a user
+    Then see the numbers of pages correctly increasing in increments of 10
```
Comment (cpauya) [inline integration_testing/features/super-admin/super-admin-find-user.feature]: Then I see the numbers of pages correctly increasing in increments of 10

File: integration_testing/features/super-admin/super-admin-export-content-to-local-drive.feature [hunk 5]
```diff
@@ -3,61 +3,68 @@ Feature: Super admin exports content to local drive
...
+        And I see the *Finished! Click "Close" button to see changes.* flag
+        And I see the *Close* button
+      When I click *Close* 
+      Then I see the *Content* page is reloaded 
+        And I open the <drive> local drive
+        And I see the *KOLIBRI_DATA* folder on the <drive> local drive
+        And I see the *content* subfolder inside 
+        And I see the *databases* and *storage* subfolders inside the *content* folder
+
+    Scenario: No writable drives found
```
Comment (cpauya) [inline integration_testing/features/super-admin/super-admin-export-content-to-local-drive.feature:63]: How can we achieve this scenario?  I haven't simulated this coz my VM always has the drive C: available. :(

**Review by cpauya — CHANGES_REQUESTED** [review]
Hi @radinamatic - need some minor changes only and this should be good to go. 😸

**Review by cpauya — APPROVED** [review]
Yey thanks - this is good to go!

File: integration_testing/features/super-admin/super-admin-export-content-to-local-drive.feature (hunk 5 above)
Comment (radinamatic) [reply integration_testing/features/super-admin/super-admin-export-content-to-local-drive.feature:63]: @cpauya Not sure how is your VM set up... On mines, when they don't have writable USB devices attached, and no shared folders configured, I can see the notification correctly.

Selection_157

Selection_158

Comment (cpauya) [reply integration_testing/features/super-admin/super-admin-export-content-to-local-drive.feature:63]: Thanks!  Got a similar config, however, in my VirtualBox 6.0.10 VM it's showing `Local Fixed Disk` even though I have no USB device attached, no Shared Folder, and no other Storage Devices aside from the SATA disk for the VM (adding a CD-ROM with an .iso attached shows up too).

Screenshot 2019-07-17 23 34 38

Will give this a pass for now and move on to other tasks. :)

Comment (radinamatic) [reply integration_testing/features/super-admin/super-admin-export-content-to-local-drive.feature:63]: Sounds good, but we should clarify it for the sake of future replicability of the tests! :+1:

**Pull request merged**